### ISSUE TITLE
[@farmfe/js-plugin-solid] during the build process, a large number of console prints appeared on the console

### ISSUE THREAD

**Issue by RSS1102** [farm-fe/farm#1204]
### Steps to reproduce

when i run `pnpm run build`,When I use `@farmfe/js-plugin-solid`, a large number of consoles appear, but when I use `vite-plugin-solid`, this error does not occur.

### What is expected?

No unexpected terminal prompts appear.

### What is actually happening?

![image](https://github.com/farm-fe/farm/assets/81673017/795a96d1-402b-4693-85e3-0668552771ac)

### System Info

```Shell
System:
    OS: Windows 11 10.0.22631
    CPU: (16) x64 AMD Ryzen 7 5825U with Radeon Graphics
    Memory: 3.05 GB / 19.86 GB
  Binaries:
    Node: 18.19.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.22 - C:\Program Files\nodejs\yarn.CMD
    npm: 10.2.3 - C:\Program Files\nodejs\npm.CMD
    pnpm: 8.11.0 - C:\Program Files\nodejs\pnpm.CMD
    bun: 1.1.2 - ~\.bun\bin\bun.EXE
  Browsers:
    Edge: Chromium (123.0.2420.97)
    Internet Explorer: 11.0.22621.1
```

### Any additional comments?

https://github.com/RSS1102/nordstjerna/tree/%40farmfe/js-plugin-solid

**Comment by wjq990112** [comment]
Sorry, I am the author of `@farmfe/js-plugin-solid`. After Farm supports the vite plugin, this JS plugin is deprecated. I will archive it as soon as possible.

**Comment by wjq990112** [comment]
May be next, I'll support SolidJS by Rust plugin.

**Comment by wjq990112** [comment]
Do you have any feature request with JS plugin?

**Comment by RSS1102** [comment]
> 抱歉，我是 的作者`@farmfe/js-plugin-solid`。 Farm支持vite插件后，这个JS插件就被弃用了。我会尽快存档。

So is it not recommended to use `@farmfe/js-plugin-solid`now?😕

**Comment by RSS1102** [comment]
> May be next, I'll support SolidJS by Rust plugin.

If there are further plans, I am happy to be involved.😃

**Comment by RSS1102** [comment]
> Do you have any feature request with JS plugin?

At present, there is none. I am currently in the early stages of solid development, and all I need is for him to successfully build solid.

**Comment by wjq990112** [comment]
> > 抱歉，我是 的作者`@farmfe/js-plugin-solid`。 Farm支持vite插件后，这个JS插件就被弃用了。我会尽快存档。
> 
> So is it not recommended to use `@farmfe/js-plugin-solid`now?😕

Yes, you can use `vite-plugin-solid` instead, it is officially supported by SolidJS.
